src/client/serialize/SConscript

```diff
@@ -0,0 +1,28 @@
+import daos_build
+
+def scons():
+    """Execute build"""
```

Comment:
(pylint-misplaced-future) __future__ import is not the first non docstring statement